crates/reporting/src/report.rs

```diff
@@ -176,6 +178,18 @@ impl<'b> Report<'b> {
         }
     }
 
+    /// Render report for the language server, where the window is narrower.
+    /// Path is not included, and the header is not emphasized with "─".
+    pub fn render_language_server(self, buf: &mut String, alloc: &'b RocDocAllocator<'b>) {
```

Comment:
Initially, I implemented this from the outside in `language_server` via a trait. However, the way we render the `load` problems is controlled by `RenderTarget`, so I needed to have `render_language_server` here.